openssl/src/x509/verify.rs

```diff
@@ -23,6 +23,41 @@ bitflags! {
     }
 }
 
+bitflags! {
+    /// Flags used to verify an `X509` certificate chain.
+    pub struct X509VerifyFlags: c_ulong {
+        const X509_V_FLAG_CB_ISSUER_CHECK = ffi::X509_V_FLAG_CB_ISSUER_CHECK;
```

Comment:
The `X509_V_FLAG_` prefix isn't necessary.

Reply:
Makes sense, went through and removed them